.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
It's SUPER convenient for multiple projects. Keeps you sane not having to change your editor each time you move projects